### DISCUSSION TITLE
Line break before and after emphasized number

### DISCUSSION THREAD

**Discussion by ad-si** [typst/typst#3717] [General]
### Description

```typst
== Heading
<label>
Lorem ipsum dolor sit amet, #strong[1 April 2024] consectetur adipiscing elit.

== Heading
<label>
Lorem ipsum dolor sit amet, #strong[\1. April 2024] consectetur adipiscing elit.

== Heading
<label>
Lorem ipsum dolor sit amet, #strong[1. April 2024] consectetur adipiscing elit.
```
yields
<img width="515" alt="image" src="https://github.com/typst/typst/assets/36796532/41ed84cc-f73b-4f25-8c18-446666b5588d">

This can't be right, can it?

### Operating system

macOS

### Typst version

- [X] I am using the latest version of Typst

**Comment by PgBiel** [comment]
`1.` is notation for the `enum` (enumerated list) element, which is a block-level element and thus interrupts the text flow. `\1.` is simply escaping that notation, thus producing the literal characters 1 and dot in the text flow. `\` produces a linebreak when followed by a space.

**Comment by laurmaedje** [comment]
While I understand that it's not great in this case, it is working as expected.
